src/Service/ActivityPub/ApHttpClient.php

```diff
@@ -205,6 +205,11 @@ private function getActorCacheKey(string $apProfileId): string
         return 'ap_'.hash('sha256', $apProfileId);
     }
 
+    private function getCollectinCacheKey(string $apAddress): string
```

Comment:
I think the function name is misspelled; “Collection” is missing the last “o”

Reply:
Solved!